libs/build_netcdf.sh

```diff
@@ -119,6 +119,8 @@ mkdir -p build && cd build
              --disable-dap \
              --enable-netcdf-4 \
              --disable-doxygen \
+             --enable-parallel4 \
+             --disable-large-file-tests \
```

Comment:
why disable large file tests? I would think we need them if we turn `MAKE_CHECK=Y` in the `config.sh`

Reply:
see above comments.